demos/JWST/S3_nirspec_fs_template.ecf

```diff
@@ -14,10 +14,14 @@ src_pos_type    gaussian    # Determine source position when not given in header
 record_ypos     True        # Option to record the y position and width for each integration (only records if src_pos_type is gaussian)
 dqmask          True        # Mask pixels with an odd entry in the DQ array
 
+# Outlier rejection along time axis
+ff_outlier      True        # Set True to use full frame; set False to use only background region
```

Comment:
Is this intentionally set to True only for the NIRSpec template? Given the risk of sigma-clipping away a deep transit, I feel like it might be better to have this be False by default. It's probably also worth putting a terse word of caution in all these ECFs as well.

Reply:
Agreed